set.mm

```diff
@@ -469438,6 +469469,445 @@ Splicing words (substring replacement)
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Order Theory
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c Monot MGalConn $.
+  $v .c_ $.
+
+  $( Define a connective symbol for use as a class variable. $)
+  c.le2 $f class .c_ $.
+
+  $( Extend class notation with monotone functions. $)
+  cmnt $a class Monot $.
+
+  $( Extend class notation with the monotone Galois connection. $)
+  cmgc $a class MGalConn $.
+
+  ${
+    $d a f v w x y $.
+    $( Define a monotone function between two ordered sets.  (Contributed by
+       Thierry Arnoux, 20-Apr-2024.) $)
+    df-mnt $a |- Monot = ( v e. _V , w e. _V |-> [_ ( Base ` v ) / a ]_
+      { f e. ( ( Base ` w ) ^m a ) | A. x e. a A. y e. a ( x ( le ` v ) y
+         -> ( f ` x ) ( le ` w ) ( f ` y ) ) } ) $.
+  $}
+
+  ${
+    $d v w a b f g x y $.
+    $( Define monotone Galois connections.  For a better readable version, see
```

Comment:
See what? There's something missing definitely.

Reply:
Right! I meant to refer to `~mgcval`.